### PR TITLE
package renaming, CI prep, and tests

### PR DESCRIPTION
**Pull request by heaper**
I didn't update the version since Aaron released v5 a couple weeks ago and no projects (DEG projects, at least) are currently using it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "moduleLoader",
+    "name": "@degjs/moduleLoader",
```
Comment (scheul93) [inline package.json]: NPM will yell at the name `moduleLoader`. It doesn't like capital letters in module names
https://docs.npmjs.com/files/package.json

Consider `module-loader` and updating README to match

**Comment by scheul93** [comment]
Do you need a .travis.yml? Or is that a file for a different day?

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+    "plugins": [
+        "@babel/plugin-syntax-dynamic-import",
```
Comment (scheul93) [inline .babelrc]: ohhhh... Babel 7? 👍

File: package.json (hunk 1 above)
Comment (heaper) [reply package.json]: Good catch, thanks. Updated.

File: .babelrc (hunk 2 above)
Comment (heaper) [reply .babelrc]: Yes indeed. It's got that new transpiler smell.

**Pull request merged**